astropy/coordinates/tests/test_solar_system.py

```diff
@@ -0,0 +1,209 @@
+    moon_astropy = _apparent_position_in_true_coordinates(moon_astropy)
+
+    assert (moon_astropy.separation(skyfield_moon) <
+            skyfield_angular_separation_tolerance)
+    assert (moon_astropy.separation_3d(skyfield_moon) < skyfield_separation_tolerance)
+
+    assert (jupiter_astropy.separation(skyfield_jupiter) <
+            skyfield_angular_separation_tolerance)
+    assert (jupiter_astropy.separation_3d(skyfield_jupiter) <
+            skyfield_separation_tolerance)
+
+    assert (mercury_astropy.separation(skyfield_mercury) <
+            skyfield_angular_separation_tolerance)
+    assert (mercury_astropy.separation_3d(skyfield_mercury) <
+            skyfield_separation_tolerance)
+
+
+@remote_data
+@pytest.mark.skipif(str('not HAS_JPLEPHEM'))
+def test_positions_distances_geocentric_1980():
+    """
+    Test positions against those generated by JPL Horizons accessed on
+    2016-03-28, with refraction turned on.
+    """
+    t = Time('1980-03-25 00:00')
+
+    frame = GCRS(obstime=t)
+
+    moon_astropy_1980 = get_moon(t)
+    mercury_astropy_1980 = get_body(t, 'mercury')
+    jupiter_astropy_1980 = get_body(t, 'jupiter')
+
+    # convert to true equator and equinox
+    jupiter_astropy_1980 = _apparent_position_in_true_coordinates(jupiter_astropy_1980)
+    mercury_astropy_1980 = _apparent_position_in_true_coordinates(mercury_astropy_1980)
+    moon_astropy_1980 = _apparent_position_in_true_coordinates(moon_astropy_1980)
+
+    # Results returned by JPL Horizons web interface
+    d_mercury_1980 = c*(6.323037*u.min)
+    d_moon_1980 = c*(0.021921*u.min)
+    d_jupiter_1980 = c*(37.694557*u.min)
+    mercury_horizons_1980 = SkyCoord(ra='22h41m47.78s', dec='-08d29m32.0s',
+                                     distance=d_mercury_1980,
+                                     frame=frame)
+    moon_horizons_1980 = SkyCoord(ra='07h32m02.62s', dec='+18d34m05.0s',
+                                  distance=d_moon_1980,
+                                  frame=frame)
+    jupiter_horizons_1980 = SkyCoord(ra='10h17m12.82s', dec='+12d02m57.0s',
+                                     distance=d_jupiter_1980,
+                                     frame=frame)
+
+    # Assert sky coordinates are close
+    assert (mercury_astropy_1980.separation(mercury_horizons_1980) <
+            separation_tolerance_planets)
+    assert (jupiter_astropy_1980.separation(jupiter_horizons_1980) <
+            separation_tolerance_planets)
+    assert (moon_astropy_1980.separation(moon_horizons_1980) <
+            separation_tolerance_moon)
+
+    # Assert distances are close
+    distances_astropy = [mercury_astropy_1980.distance,
+                         moon_astropy_1980.distance,
+                         jupiter_astropy_1980.distance]
+    distances_horizons = [d_mercury_1980, d_moon_1980, d_jupiter_1980]
+
+    assert_quantity_allclose(distances_astropy, distances_horizons,
+                             rtol=distance_tolerance)
```

Comment:
This should be `atol` not `rtol`! (and then you can use the unit on distance tolerance).